### PR TITLE
[Spark load][Document] Add docs about spark and yarn client for spark load

### PR DESCRIPTION
**Pull request by xy720**
## Proposed changes

Add docs about spark and yarn client for spark load

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: docs/zh-CN/administrator-guide/load-data/spark-load-manual.md [hunk 1]
```diff
@@ -229,7 +230,43 @@ GRANT USAGE_PRIV ON RESOURCE * TO ROLE "role0";
 REVOKE USAGE_PRIV ON RESOURCE "spark0" FROM "user0"@"%";
 ```
 
+### 配置 SPARK 客户端
+
+FE底层通过执行spark-submit的命令去提交spark任务，因此需要为FE配置spark客户端，建议使用2.4以上的spark2官方版本，[spark下载地址](https://archive.apache.org/dist/spark/)，下载完成后，请按步骤完成以下配置。
```
Comment (morningman) [inline docs/zh-CN/administrator-guide/load-data/spark-load-manual.md:235]: 可以给一个具体的版本号，比如2.4.3

File: docs/zh-CN/administrator-guide/load-data/spark-load-manual.md [hunk 2]
```diff
@@ -229,7 +230,43 @@ GRANT USAGE_PRIV ON RESOURCE * TO ROLE "role0";
...
+    |        |-__lib_1bbb74bb6b264a270bc7fca3e964160f_spark-2x.zip
+    |-__archive_1.2.0/
+    |        |-...
+```
+
+除了spark依赖(默认以`spark-2x.zip`命名)，FE还会上传DPP的依赖包至远端仓库，若此次spark load提交的所有依赖文件都已存在远端仓库，那么就不需要在上传依赖，省下原来每次重复上传大量文件的时间。
 
+### 配置 YARN 客户端
+
+FE底层通过执行yarn命令去获取正在运行的application的状态以及杀死application，因此需要为FE配置yarn客户端，建议使用2.5以上的hadoop官方版本，[hadoop下载地址](https://archive.apache.org/dist/hadoop/common/)，下载完成后，请按步骤完成以下配置。
```
Comment (morningman) [inline docs/zh-CN/administrator-guide/load-data/spark-load-manual.md:263]: 可以给一个具体版本号

**Review by morningman — COMMENTED** [review]
You need to rebase the code

File: docs/zh-CN/administrator-guide/load-data/spark-load-manual.md (hunk 1 above)
Comment (xy720) [reply docs/zh-CN/administrator-guide/load-data/spark-load-manual.md]: ok,done.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**